Non-profits.md

```diff
@@ -5,6 +5,16 @@ There are also some non-profits that share some or all of their code.
 * [Wikipedia - Wikimedia](https://github.com/wikimedia)
 
 
+## [Gene Ontology consortioum] (https://github.com/geneontology/)
```

Comment:
I am not sure if that space between the closing square bracket and the opening parentheses won't cause a problem. I'll accept the PR and we'll see it on the web version of the file.